src/test/java/com/egov/business-services/constants/billing-service-demand.yaml

```diff
@@ -0,0 +1,22 @@
+parameters:
+  consumerCode: PT_test_
+  consumerType: PropertyTax
+  businessService: PT
+  taxHeadMasterCode: PT_TAX
+  taxAmount: 200
+  collectionAmount: 0
+  minimumAmountPayable: 1
+  status: ACTIVE
+invalidParameters:
+  businessService: invalid
+  taxPeriodFrom: 11111111111111111111111111111111111111111111
+  taxHeadMasterCode: invalid
+  taxAmountGreater: 20020020012
+  minimumAmountPayableGreater: 10000000000
+expectedMessages:
+  duplicateConsumerCode: "Demand already exists in the same period with the same businessService for the given consumercodes : "
+  invalidBusinessService: "No Business Service masters found for give codes "
+  zeroTaxPeriodError: "No Tax Periods Found for the given demand with fromDate : 0 and toDate : 0"
+  invalidTaxHeadMasterCode: "No TaxHead masters found for give codes "
+  collectionAmountMoreThanTaxAmount: cannot not be greater than taxAmount
+  
```

Comment:
We already have `billing-services.yaml` file in our framework, so can you move these params and messages there itself?

Reply:
billing service demand is a subset of billing service. some parameter values also change. ex: here i'm using consumer code and appending a random string dynamically.